Disaster-Response-Platform/mobile/DisasterResponsePlatform/app/src/main/java/com/example/disasterresponseplatform/models/CredibleUser.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.disasterresponseplatform.models
```

Comment:
Could you please move this package under the data -> models -> user types ( you can create this package ) to continue our package design